packages/outline-templates/default/scripts/link-for-local-dev.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Takes an argument of the path to a directory containing package directories to link for
+ * local development. Will register them all for local development in the current project.
+ */
```

Comment:
Now that I'm looking through things, I'm thinking running `yarn turbo dev` does this essential same linking of the local packages. 

I've not used the mode but @grshane reminded me it was a thing.

Reply:
So I'm pondering if we can be running `yarn turbo dev` and then run the CLI commands in a similar way.. .

Reply:
This script is intended for use in a generated project. I don't think `yarn turbo dev` would apply in that context. Am I overlooking something?

Reply:
Derp. In my head this was for Outline local dev.

Reply:
This is what I was thinking of where `yarn turbo run dev` would possibly serve the same purpose as what's going on there. https://github.com/phase2/outline/actions/runs/5046433815/jobs/9052038359#step:7:15

It's worth some digging later.